libs/user/user.ts

```diff
@@ -74,13 +74,15 @@ class UserFactory {
 
     const {
       clientApiUrl,
+      clientApiTxUrl,
+      clientApiBpUrl,
       blockchainWsUrl,
       ethereumPrivateKey: ethPrivateKey,
       nightfallMnemonic,
     } = value;
 
-    // Instantiate Client
-    const client = new Client(clientApiUrl);
+    // Instantiate Client and optional Workers URL
+    const client = new Client(clientApiUrl, clientApiTxUrl, clientApiBpUrl);
```

Comment:
I like your comments, but can you please update 1 more place?
In this file, on top of `create` function:

```
* @param {string} options.clientApiUrl HTTP URL of a running Nightfall Client
* @param {string} [options.clientApiTxUrl] HTTP URL of a running Transaction Handler Worker
* @param {string} [options.clientApiBpUrl] HTTP URL of a running Block Proposed Worker
```

The brackets `[]` mean optional